applications/StructuralMechanicsApplication/tests/formfinding_test/Fofi_4Point_Tent_noCable_test_parameters.json

```diff
@@ -4,14 +4,14 @@
         "model_part_name" : "Structure",
         "domain_size"     : 3,
         "parallel_type"   : "OpenMP",
-        "time_step"       : 1.0,
+        "time_step"       : 1.1,
         "start_time"      : 0.0,
-        "end_time"        : 10.0,
-        "echo_level"      : 0
+        "end_time"        : 1.0,
+        "echo_level"      : 2
     },
     "solver_settings"          : {
         "solver_type"                        : "Static",
-        "echo_level"                         : 0,
+        "echo_level"                         : 2,
```

Comment:
Please set the echo_level to 0 in all of your tests

Reply:
done